flink/v1.13/build.gradle

```diff
@@ -40,6 +40,8 @@ project(':iceberg-flink:iceberg-flink-1.13') {
     implementation project(':iceberg-parquet')
     implementation project(':iceberg-hive-metastore')
 
+    implementation "com.github.ben-manes.caffeine:caffeine"
```

Comment:
Please change just the latest Flink version. We'll backport the changes after they are reviewed and committed. That avoids needing to review the same code 3 times.

Reply:
Sure, I will only modify it for the latest version first